js/actions/index.ts

```diff
@@ -61,75 +64,56 @@ export function fetchAndObserveData() {
 
 function receivePortalData(rawPortalData: IPortalRawData) {
   return (dispatch: Dispatch) => {
+    firestoreInitialized.then(db => _receivePortalData(db, rawPortalData, dispatch));
+  };
+}
+
+function _receivePortalData(db: firebase.firestore.Firestore,
+                            rawPortalData: IPortalRawData, dispatch: Dispatch) {
+  dispatch({
+    type: RECEIVE_PORTAL_DATA,
+    response: rawPortalData
+  });
+  let resourceUrl = rawPortalData.offering.activity_url.toLowerCase();
+  if (resourceUrl.match(/http:\/\/.*\.concord\.org/)) {
```

Comment:
We have authoring.staging.concord.org though so this wouldn't match it correctly. 
But we could try:
`/http:\/\/[^=?&]+\.concord\.org/`

Or we could limit it just to authoring with something like:
`/http:\/\/authoring(\.staging)?\.concord\.org/`
(I'm not sure if I've got that syntax right though)

But those are also fragile. We have authoring.concordqa.org too so we'd have to add a check for that.  

I think the best thing to do here is to drop the conditional and convert all urls to https. This will break some dev existing environments, but they can be updated to use https and republish their resources. This URL should just be used for looking up stuff in firestore. So https://localhost or https://app.lara.docker should work fine even if the resources aren't available there. I think that should be a separate PR though, since this PR is just keeping the status quo for this logic and its intention is to change other things.

Reply:
Moving to a separate story sounds good :+1: